server/vulnerabilities/sync.go

```diff
@@ -202,6 +202,7 @@ func LoadCVEMeta(vulnPath string, ds fleet.Datastore) error {
 	}
 
 	for _, epssScore := range epssScores {
```

Comment:
Static check where are you when we need you? :) (taking the address of the loop variable...)